andela/travel_tool_front
# #161545272 Reformat error message for failed validation

#### What does this PR do?
This fixes the error messages that are displayed during form validation when a user inputs the wrong thing.For example when the expected error message is:

> Name should be typed

the actual error displayed is:

> Bad request.Name should be typed

#### Description of Task to be completed?


#### How should this be manually tested?
1. Clone the repo - git clone https://github.com/andela/travel_tool_front.git
2. Checkout this branch - ```git checkout bg-reformat-error-message-#161545272```
3. Run ```yarn start```
2. Login to the application as a super admin.
4. Navigate to trip planner page
5. Fill in white space in the create checklist form
6. Click add item

The error message displayed should be `Name is required`
and not `Bad Request.Name is required`.
This can also be observed on other forms that requires validation.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?

[#161545272]()
#### Screenshots (if appropriate)
before:
<img width="765" alt="screen shot 2018-11-23 at 15 29 51" src="https://user-images.githubusercontent.com/19623571/48943618-cd823300-ef34-11e8-9f5c-ae72661426c4.png">
after:
<img width="910" alt="screen shot 2018-11-23 at 15 28 19" src="https://user-images.githubusercontent.com/19623571/48943633-d246e700-ef34-11e8-8549-f2f508fa2521.png">

#### Questions:
